iceoryx_systemtest/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 Apex.AI, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
```

Comment:
Very good! The only thing missing is the SPDX license statement at the end of the fileheader. I know you will bite into your keyboard and wish me all the best, but to cite a famous captain:
> It is possible to commit no mistakes and still lose. That is not a weakness—that is life.
—”Peak Performance” (TNG S02 Ep. 21)

example:
```cpp
    // See the License for the specific language governing permissions and
    // limitations under the License.
    //
    // SPDX-License-Identifier: Apache-2.0
````

@dkroenke Please image a quiet but annoying evil laughter in your head (with the voice of Morgan Freeman)

Reply:
**Engage!**